homeassistant/components/frontend/www_static/micromarkdown-js.html

```diff
@@ -0,0 +1,244 @@
+.mmd_rss,
+.mmd_feed,
+.mmd_link,
+.mmd_shopping,
+.mmd_quote,
+.mmd_glossary,
+.mmd_podcast,
+.mmd_video,
+.mmd_wiki,
+.mmd_arduinocc,
+.mmd_amazon,
+.mmd_applecom,
+.mmd_appnet,
+.mmd_facebookcom,
+.mmd_flattr,
+.mmd_flattrcom,
+.mmd_twitter,
+.mmd_tweet,
+.mmd_twittercom,
+.mmd_wikipediaorg,
+.mmd_youtube,
+.mmd_youtubecom,
+.mmd_googlede,
+.mmd_googlecom,
+.mmd_google,
+.mmd_heisede,
+.mmd_amazonde,
+.mmd_amazoncom,
+.mmd_paypalcom,
+.mmd_esaint,
+.mmd_dlrde,
+.mmd_wiktionaryorg,
+.mmd_deviantartcom,
+.mmd_podloveorg,
+.mmd_dropboxcom,
+.mmd_nytimescom,
+.mmd_shownotes,
+.mmd_githubcom,
+.mmd_imdbcom,
+.mmd_flickrcom,
+.mmd_tumblrcom,
+.mmd_threemach,
+.mmd_soundcloudcom,
+.mmd_slashdotorg,
+.mmd_stackoverflow,
+.mmd_yahoocom,
+.mmd_xingcom,
+.mmd_xenimde,
+.mmd_crefm,
+.mmd_einschlafenpodcastde,
+.mmd_podunioncom,
+.mmd_1337kulturde,
+.mmd_binaergewitterde,
+.mmd_nasagov,
+.mmd_diesondersendungde,
+.mmd_wikigeeksde,
+.mmd_helmholtzde,
+.mmd_famousprojectorg,
+.mmd_nsagov,
+.mmd_thenextwebcom,
+.mmd_achtungabschriftde,
+.mmd_auphoniccom,
+.mmd_creativecommonsorg,
+.mmd_cccde,
+.mmd_raspberrypiorg,
+.mmd_golemde,
+.mmd_selfcssorg,
+.mmd_firtzorg,
+.mmd_hoersuppede,
+.mmd_raumzeitpodcastde,
+.mmd_absradiode,
+.mmd_staatsbuergerkundepodcastde,
+.mmd_openstreetmapde,
+.mmd_openstreetmaporg,
+.mmd_tagesschaude,
+.mmd_uberspacede,
+.mmd_wrintde {
+  background-repeat: no-repeat;
+  background-position: 0 center;
+  padding-left: 18px;
+  margin-left: 4px;
+  background-size: 16px 16px;
+}
+
+.mmd_podcast { background-image: url('https://raw2.github.com/SimonWaldherr/BitmapWebIcons/gh-pages/podcast.png'); }
```

Comment:
This one and all below do not seem to be related to micromarkdown.js. Also the links no longer work either.

After removing this, make sure you regenerate the gzip file to keep it up to date.